cmd/kpod/spec.go

```diff
@@ -17,6 +17,33 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func blockAccessToKernelFilesystems(config *createConfig, g *generate.Generator) {
+	if !config.privileged {
+		for _, mp := range []string{
+			"/proc/kcore",
+			"/proc/latency_stats",
+			"/proc/timer_list",
+			"/proc/timer_stats",
+			"/proc/sched_debug",
+			"/proc/scsi",
+			"/sys/firmware",
+		} {
+			g.AddLinuxMaskedPaths(mp)
+		}
+
+		for _, rp := range []string{
+			"/proc/asound",
+			"/proc/bus",
+			"/proc/fs",
+			"/proc/irq",
+			"/proc/sys",
+			"/proc/sysrq-trigger",
```

Comment:
should sqsrq-trigger be read-only?  It seems to have only write perms for the root user on my machines.

Reply:
Not sure, this code was taken from CRI-O